myExpenses/src/main/java/org/totschnig/myexpenses/fragment/TransactionList.kt

```diff
@@ -48,6 +58,33 @@ class TransactionList : BaseTransactionList() {
         }
     }
 
+    override fun setTitle(mode: ActionMode, lv: AbsListView, position: Int, checked: Boolean) {
+        mTransactionsCursor.moveToPosition(position)
+        val cost = mTransactionsCursor.getLong(mTransactionsCursor.getColumnIndex(KEY_AMOUNT))
+        getSum(cost, checked)
```

Comment:
It would be cleaner if changing of the sum would not be done as a side effect in method setTitle, but explicitly in a special method, called "onSelectionChanged" for example.